pyxem/signals/__init__.py

```diff
@@ -75,7 +75,7 @@ def select_method_from_method_dict(method, method_dict, **kwargs):
         raise NotImplementedError("The method `{}` is not implemented. "
                                   "See documentation for available "
                                   "implementations.".format(method))
-    elif not kwargs:
+    elif print_help:
```

Comment:
you weren't to know, but this should be `elif print_help and not kwargs` because of how other bits of the code base behave.